### DISCUSSION TITLE
Thuật ngữ hard/soft margin, kernel(ized)

### DISCUSSION THREAD

**Discussion by quangvnai** [mlbvn/handson-ml2-vn#33] [Terminology] [answered]
Mình gặp một số từ không biết có thể dịch được luôn không?

* `hard margin` -> `biên cứng`
* `soft margin` -> `biên mềm`
* `kernel trick` -> `kĩ thuật hạt nhân`?
*  `kernelized SVM` -> `SVM hạt nhân hoá`?
Mọi người cho ý kiến nhé. @mlbvn/collaborators

**Comment by minhduc0711** [comment, marked as answer]
mình copy từ sách anh Tiệp :v
* `kernel trick` -> `thủ thuật hạt nhân`
* `kernelized SVM` -> `SVM hạt nhân`
2 từ còn lại thì ok

**Reply by quangvnai** [reply]
Ok, đồng chí nào review nhớ check đoạn này nhé nếu mình có quên.
